.env.example

```
@@ -1,18 +1,28 @@
 # Copy this file to .env before starting the local stack.
 
-# Local database startup
-POSTGRES_DB=safequery
-POSTGRES_USER=safequery
-POSTGRES_PASSWORD=change-me-for-shared-environments
+# Local application PostgreSQL startup
+APP_POSTGRES_DB=safequery
+APP_POSTGRES_USER=safequery
+APP_POSTGRES_PASSWORD=change-me-for-shared-environments
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The APP_POSTGRES_PASSWORD key should go before the APP_POSTGRES_USER key

(UnorderedKey)